shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ShortCircuitConstants.java

```diff
@@ -11,6 +11,17 @@
  */
 public final class ShortCircuitConstants {
 
+    public enum VoltageMapType {
+        NOMINAL,
+        PREVIOUS
+    }
+
+    public enum NominalVoltageMapType {
+        IEC_909,
+        RTE,
```

Comment:
You cannot mention `RTE` here! What is the rule of `RTE`? What is the rule of `IEC_909` (add a link?)?